x/wasm/simulation/operations.go

```diff
@@ -0,0 +1,161 @@
+		weightMsgInstantiateContract int
+	)
+
+	simstate.AppParams.GetOrGenerate(simstate.Cdc, OpWeightMsgStoreCode, &weightMsgStoreCode, nil,
+		func(_ *rand.Rand) {
+			weightMsgStoreCode = params.DefaultWeightMsgStoreCode
+		},
+	)
+
+	simstate.AppParams.GetOrGenerate(simstate.Cdc, OpWeightMsgInstantiateContract, &weightMsgInstantiateContract, nil,
+		func(_ *rand.Rand) {
+			weightMsgInstantiateContract = params.DefaultWeightMsgInstantiateContract
+		},
+	)
+
+	return simulation.WeightedOperations{
+		simulation.NewWeightedOperation(
+			weightMsgStoreCode,
+			SimulateMsgStoreCode(ak, bk, wasmKeeper),
+		),
+		simulation.NewWeightedOperation(
+			weightMsgInstantiateContract,
+			SimulateMsgInstantiateContract(ak, bk, wasmKeeper),
+		),
+	}
+}
+
+// SimulateMsgStoreCode generates a MsgStoreCode with random values
+func SimulateMsgStoreCode(ak types.AccountKeeper, bk simulation.BankKeeper, wasmKeeper WasmKeeper) simtypes.Operation {
+	return func(
+		r *rand.Rand,
+		app *baseapp.BaseApp,
+		ctx sdk.Context,
+		accs []simtypes.Account,
+		chainID string,
+	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
+		if wasmKeeper.GetParams(ctx).CodeUploadAccess.Permission != types.AccessTypeEverybody {
+			return simtypes.NoOpMsg(types.ModuleName, types.MsgStoreCode{}.Type(), "no chain permission"), nil, nil
+		}
+
+		config := &types.AccessConfig{
+			Permission: types.AccessTypeEverybody,
+		}
+
+		simAccount, _ := simtypes.RandomAcc(r, accs)
+		msg := types.MsgStoreCode{
+			Sender:                simAccount.Address.String(),
+			WASMByteCode:          reflectContract,
+			InstantiatePermission: config,
+		}
+
+		txCtx := simulation.OperationInput{
+			R:             r,
+			App:           app,
+			TxGen:         simappparams.MakeTestEncodingConfig().TxConfig,
+			Cdc:           nil,
+			Msg:           &msg,
+			MsgType:       msg.Type(),
+			Context:       ctx,
+			SimAccount:    simAccount,
+			AccountKeeper: ak,
+			Bankkeeper:    bk,
+			ModuleName:    types.ModuleName,
+		}
+
+		return simulation.GenAndDeliverTxWithRandFees(txCtx)
+	}
+}
+
+// SimulateMsgInstantiateContract generates a MsgInstantiateContract with random values
+func SimulateMsgInstantiateContract(ak types.AccountKeeper, bk simulation.BankKeeper, wasmKeeper WasmKeeper) simtypes.Operation {
+	return func(
+		r *rand.Rand,
+		app *baseapp.BaseApp,
+		ctx sdk.Context,
+		accs []simtypes.Account,
+		chainID string,
+	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
+		simAccount, _ := simtypes.RandomAcc(r, accs)
+
+		var contractInfo types.ContractInfo
+		wasmKeeper.IterateContractInfo(ctx, func(address sdk.AccAddress, info types.ContractInfo) bool {
+			contractInfo = info
+			return true
+		})
+
+		if contractInfo.Equal(types.ContractInfo{}) {
+			return simtypes.NoOpMsg(types.ModuleName, types.MsgInstantiateContract{}.Type(), "no contracts available"), nil, nil
+		}
+
+		spendable := bk.SpendableCoins(ctx, simAccount.Address)
```

Comment:
Very nice to send some tokens. 💸 
Test accounts will get broke fast though. How about sending a random amount with [simtypes.RandSubset](https://github.com/cosmos/cosmos-sdk/blob/v0.45.4/x/bank/simulation/operations.go#L406) as the sdk does?